src/prefect/config.toml

```diff
@@ -99,7 +99,7 @@ datefmt = "%Y-%m-%d %H:%M:%S"
 log_to_cloud = false
 
 # Extra loggers for Prefect log configuration
-extra_loggers = "[]"
+extra_loggers = "['py.warnings']"
```

Comment:
Perhaps a boolean flag would make more sense than adding it to `extra_loggers`? It also makes the option more discoverable if not enabled by default, since the option would still appear in the `config.toml`.

Reply:
Yea that makes sense; I originally wanted to avoid adding another config setting for something somewhat minor like this.